cg/meta/clean/demultiplexed_flow_cells.py

```diff
@@ -53,6 +59,13 @@ def __init__(
         self._files_exist_in_housekeeper = None
         self._files_exist_on_disk = None
         self._passed_check = None
+        self._sequencer_type = None
+
+    @property
+    def sequencer_type(self) -> str:
```

Comment:
```suggestion
    def sequencer_type(self) -> Optional[str]:
```
I do not know if this is the proper way of writing this as the function can also return None. I guess an alternative can be Union[None, str]. I think mypy also like explicit returns - so adding a return [VALUE] after the if condition

Reply:
I prefer the notation using Optional